build.cmd

```diff
@@ -22,49 +21,54 @@ if /I "%1" == "/no-multi-proc" set MSBuildAdditionalArguments=&&shift&& goto :Pa
 call :Usage && exit /b 1
 :DoneParsing
 
-if not exist "%DeveloperCommandPrompt%" (
-  echo In order to build this repository, you need Visual Studio "15" Preview installed.
+if not exist "%VS150COMNTOOLS%" (
+  echo To build this repository, this script needs to be run from a Visual Studio 2017 RC developer command prompt.
   echo.
-  echo Visit this page to download:
+  echo If Visual Studio is not installed, visit this page to download:
   echo.
-  echo http://go.microsoft.com/fwlink/?LinkId=746567
+  echo https://www.visualstudio.com/vs/visual-studio-2017-rc/
   exit /b 1
 )
 
 if not exist "%VSSDK150Install%" (
-  echo In order to build this repository, you need to modify your Visual Studio installation to include "Visual Studio Extensibility Tools".
+  echo To build this repository, you need to modify your Visual Studio installation to include the "Visual Studio extension development" workload.
   exit /b 1
 )
 
-call "%DeveloperCommandPrompt%" || goto :BuildFailed
```

Comment:
Again, are we sure that not running VsDevCmd.bat will not break microbuild? Tagging @basoundr